afs/afs-core/src/test/java/com/powsybl/afs/AfsBaseTest.java

```diff
@@ -204,4 +204,18 @@ public void moveToTest() {
         assertEquals(1, test2.getChildren().size());
     }
 
+    @Test
+    public void findProjectFolderTest() {
+
+        Project project = afs.getRootFolder().createProject("test");
+        ProjectFolder test1 = project.getRootFolder().createFolder("test1");
+        ProjectFolder projectFolderResultat = afs.findProjectFolder(test1.getId());
+
+        assertNotNull(projectFolderResultat);
+        assertEquals(projectFolderResultat.getId(), test1.getId());
```

Comment:
Two parts of assertions must be inverted ; test framework defines assertAquals arguments as:
assertEquals(Object expected, Object actual)